WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderNavigationButton.swift

```diff
@@ -30,7 +30,11 @@ struct ReaderNavigationButton: View {
             if let selectedItem = viewModel.selectedItem {
                 menuLabel(for: selectedItem)
             }
-        }.buttonStyle(PlainButtonStyle())
+        }
+        .buttonStyle(PlainButtonStyle())
+        .onTapGesture {
+            WPAnalytics.track(.readerDropdownOpened)
```

Comment:
Hmm, the only issue is that this event gets triggered again when tapping on the button while the menu is being presented. But I guess it'll be a low number of occurrences, hopefully. I'm not sure if there's a trivial way to detect if a menu is currently being presented, though.

Reply:
Yeah, I tried to group the button elements and use an `onAppear` but it only triggered once. The only other thing I can think of is creating our own menu UI, but that doesn't seem worth the effort to me. We'll still have the tracks from when the menu buttons are actually tapped, so I don't think it'll be a big deal if it's slightly incorrect.